src/common/helpers/github-url.js

```diff
@@ -2,6 +2,12 @@ import parseGitHubUrl from 'parse-github-url';
 
 export const getGitHubRepoUrl = (owner, name) => {
   let repository = !name ? owner : `${owner}/${name}`;
+
+  // if already a valid GH repository
+  if (repository.indexOf('https://github.com/') === 0) {
```

Comment:
Sometimes we pass the redux store 'id' to this function. In some legacy places (like register name actions) this id is repo fullName (so the function turns it into URL).

With our new convention the id in store (for the snap) is full repo URL, so the parts of code that pass the id here now pass the full URL.

In theory it may be a temporary thing until we have ids as full URLs everywhere for snaps in store, so the transformation would not be needed.